fulltext/routes.py

```diff
@@ -66,6 +64,7 @@ def retrieve(paper_id: str, version: Optional[str] = None,
     if content_type == 'text/plain':
```

Comment:
Looked up `Content-Type: text/plain; charset=utf-8` which is a possibility but isn't used